README.txt

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Readme.txt is basically superceeded by README.md. Do I need both or is just the markdown OK?